packages/client/src/api.ts

```diff
@@ -324,7 +324,7 @@ async function executeServerAbility(
  */
 export async function executeAbility(
 	name: string,
-	input: AbilityInput = null
+	input: AbilityInput
```

Comment:
This would make the `input` mandatory while it isn't. 
```suggestion
	input?: AbilityInput
```

I'm not an TS expert, but it feels like this would allow the value to be undefined.